data/api/client-server/admin.yaml

```diff
@@ -41,75 +30,82 @@ paths:
-                      items:
-                        type: object
-                        title: SessionInfo
-                        properties:
-                          connections:
-                            type: array
-                            description: Information particular connections in the session.
-                            items:
-                              type: object
-                              title: ConnectionInfo
-                              properties:
-                                ip:
-                                  type: string
-                                  description: Most recently seen IP address of the session.
-                                last_seen:
-                                  type: integer
-                                  format: int64
-                                  description: Unix timestamp that the session was last active.
-                                user_agent:
-                                  type: string
-                                  description: User agent string last seen in the session.
+                properties:
+                  user_id:
+                    type: string
+                    description: The Matrix user ID of the user.
+                  devices:
+                    type: object
+                    description: Each key is an identifier for one of the user's devices.
+                    additionalProperties:
+                      type: object
+                      title: DeviceInfo
+                      properties:
+                        sessions:
+                          type: array
+                          description: A user's sessions (i.e. what they did with an access token from one
+                            login).
+                          items:
+                            type: object
+                            title: SessionInfo
+                            properties:
+                              connections:
+                                type: array
+                                description: Information particular connections in the session.
+                                items:
+                                  type: object
+                                  title: ConnectionInfo
+                                  properties:
+                                    ip:
+                                      type: string
+                                      description: Most recently seen IP address of the session.
+                                    last_seen:
+                                      type: integer
+                                      format: int64
+                                      description: Unix timestamp that the session was last active.
+                                    user_agent:
+                                      type: string
+                                      description: User agent string last seen in the session.
+              examples:
```

Comment:
Most of JSON examples were converted to YAML during conversion. Should we do that for all examples? Revert this to make the diff smaller?

Reply:
As far as I understand, the script parses everything, edits it and then serializes it according to the file extension, that's why comments are gone and everything is converted to YAML.

One thing we could do is to see if it's possible to copy examples and comments and link them to a field, and we reintroduce them in the proper place after the conversion.

Reply:
> As far as I understand, the script parses everything, edits it and then serializes it according to the file extension, that's why comments are gone and everything is converted to YAML.

Makes sense. I don't think it's worth spending a lot of effort to work around this.